perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -136,8 +145,10 @@ def _Exists(self):
     """
     cmd = util.GcloudCommand(self, 'sql', 'instances', 'describe',
                              self.instance_id)
-    _, _, retcode = cmd.Issue()
-    return retcode == 0
+    stdout, _, _ = cmd.Issue()
+    json_output = json.loads(stdout)
```

Comment:
if the command fails, then most likely the stdout is empty or it would not be a JSON result, in that case, we should not try to do json.loads; alternatively, we can always do json.loads here but wrap it in try_except. On any exception, we consider the call failed and return "not exist".